src/foam/lib/formatter/AbstractFObjectFormatter.java

```diff
@@ -116,6 +118,32 @@ protected synchronized List getProperties(ClassInfo info) {
     return propertyMap_.get(of);
   }
 
+  protected synchronized List getDelta(FObject oldFObject, FObject newFObject) {
+    ClassInfo info           = oldFObject.getClassInfo();
+    String    of             = info.getObjClass().getSimpleName();
```

Comment:
Why not call toLowerCase() here to avoid calling for each property?

Reply:
cut and paste, I'll fix where it came from as well.

Reply:
fixed.  Also in getProperties.